src/components/shared/field/field.jsx

```diff
@@ -0,0 +1,47 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React, { forwardRef } from 'react';
+
+export const FIELD_TAGS = {
+  INPUT: 'input',
+  TEXTAREA: 'textarea',
+};
+
+const Field = forwardRef(({ fieldName, className, tag: Tag, error, ...otherProps }, ref) => (
+  <div className="flex flex-col w-full">
```

Comment:
```suggestion
  <div className="relative flex flex-col w-full">
```

let's set the wrapper to `position: relative`, so that we can position the errors relative to Field

<img width="350" alt="Screenshot 2022-03-25 at 13 59 56" src="https://user-images.githubusercontent.com/17677196/160110027-9ecd3da7-d81a-461e-81f8-40dea89ab9f0.png">